### ISSUE TITLE
CSS: body > div { padding: 10px } breaks scrollbar in Chrome

### ISSUE THREAD

**Issue by breunigs** [ajaxorg/ace#1202]
To see the issue, modify the example given in the embedding guide[1] like this:
1. Add CSS: `body > div { padding: 10px }`
2. Wrap the `<div id="editor>` in another `div`, `span` or other element of your choice.

The CSS rule doesn’t apply to ace, yet the vertical scrollbar is not displayed anymore. If you inspect the page the `.ace_sb` element is set to `width: 0`. Overwriting this works, but then the content is too wide. It seems as if the above CSS rule breaks calculations in Chrome.

It works with Iceweasel 18, Opera 12.12.
Breaks in Chromium 22.0.1229.94

[1] http://ace.ajax.org/#nav=embedding
